spec.bs

```diff
@@ -3048,6 +3140,20 @@ dictionary ReportWinBrowserSignals : ReportingBrowserSignals {
     bidding signals server
 </dl>
 
+<xmp class="idl">
+dictionary DirectFromSellerSignalsForBuyer {
+  DOMString auctionSignals;

```

Comment:
Shouldn't these be `any` instead of `DOMString`? Ditto for the others. We need to convert the strings back to JS objects when passing to worklets.

Reply:
Modified again after understanding what's passed in to worklets.